holmes-py/specs/gdc_data_portal_v2/analysis_center/tools_text.spec

```diff
@@ -23,7 +23,7 @@ tags: gdc-data-portal-v2, navigation, analysis-center
   |Age at Diagnosis |Apply                |
 * Wait for cohort bar case count loading spinner
 * Pause "2" seconds
-* Navigate to "Analysis" from "Header" "section"
+* Navigate to Analysis Center via URL
```

Comment:
I'm not too keen on changing this. This deviates us from using BDD format. Let me see if I can try to find an alternative.

Reply:
Put back and made additional waits